src/WatchDogs.Contracts/ICurrencyBucket.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WatchDogs.Contracts;
+public interface ICurrencyBucket
```

Comment:
What is the benefit of the interface? We are not using polymorphism here and it does not hide any behavior. `CurrencyBucket` is only a data class

Reply:
IDK. I was told to hide everything behind interfaces but i will modify it immediately.